docs/user-guide/running.md

```
@@ -577,76 +584,133 @@ default in both the CLI and SDK. PII on by default means no config flag is neede
     ```
 
-    The SDK builder merges partial overrides with
-    [`PiiReplacerConfig.get_default_config()`][nemo_safe_synthesizer.config.replace_pii.PiiReplacerConfig], so you don't need to
-    provide the full `steps` list.
+    Partial overrides merge with the defaults of
+    [`PiiReplacerConfig`][nemo_safe_synthesizer.config.replace_pii.PiiReplacerConfig],
+    so you only pass the fields you are changing.
 
 === "Config reference"
 
     ```yaml
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)